list_remaining_issues.py

```
@@ -0,0 +1,6 @@
+import subprocess
+
+def run_cmd(cmd):
+    return subprocess.check_output(cmd, shell=True, text=True)
```

Ruff (0.15.5)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)